public_html/planeObject.js

```diff
@@ -300,30 +296,80 @@ PlaneObject.prototype.getMarkerColor = function() {
         return 'hsl(' + (h/5).toFixed(0)*5 + ',' + (s/5).toFixed(0)*5 + '%,' + (l/5).toFixed(0)*5 + '%)'
 }
 
+PlaneObject.prototype.getAltitudeColor = function(altitude) {
+        var h, s, l;
+
+        if (typeof altitude === 'undefined') {
+            altitude = this.altitude;
+        }
+
+        if (altitude === null) {
+                h = ColorByAlt.unknown.h;
+                s = ColorByAlt.unknown.s;
+                l = ColorByAlt.unknown.l;
+        } else if (this.altitude === "ground") {
+                h = ColorByAlt.ground.h;
+                s = ColorByAlt.ground.s;
+                l = ColorByAlt.ground.l;
+        } else {
+                s = ColorByAlt.air.s;
+                l = ColorByAlt.air.l;
+
+                // find the pair of points the current altitude lies between,
+                // and interpolate the hue between those points
+                var hpoints = ColorByAlt.air.h;
+                h = hpoints[0].val;
+                for (var i = hpoints.length-1; i >= 0; --i) {
+                        if (altitude > hpoints[i].alt) {
+                                if (i == hpoints.length-1) {
+                                        h = hpoints[i].val;
+                                } else {
+                                        h = hpoints[i].val + (hpoints[i+1].val - hpoints[i].val) * (altitude - hpoints[i].alt) / (hpoints[i+1].alt - hpoints[i].alt)
+                                }
+                                break;
+                        }
+                }
+        }
+
+         if (h < 0) {
+                h = (h % 360) + 360;
+        } else if (h >= 360) {
+                h = h % 360;
+        }
+
+        if (s < 5) s = 5;
+        else if (s > 95) s = 95;
+
+        if (l < 5) l = 5;
+        else if (l > 95) l = 95;
+
+        return [h, s, l];
+}
+
 PlaneObject.prototype.updateIcon = function() {
         var scaleFactor = Math.max(0.2, Math.min(1.2, 0.15 * Math.pow(1.25, ZoomLvl))).toFixed(1);
 
         var col = this.getMarkerColor();
         var opacity = 1.0;
         var outline = (this.position_from_mlat ? OutlineMlatColor : OutlineADSBColor);
         var baseMarker = getBaseMarker(this.category, this.icaotype, this.typeDescription, this.wtc);
-        var weight = ((this.selected && !SelectedAllPlanes ? 2 : 1) / baseMarker.scale / scaleFactor).toFixed(1);
+        //var weight = ((this.selected && !SelectedAllPlanes ? 2 : 1) / baseMarker.scale / scaleFactor).toFixed(1);
         var rotation = (this.track === null ? 0 : this.track);
-        var transparentBorderWidth = (32 / baseMarker.scale / scaleFactor).toFixed(1);
+        //var transparentBorderWidth = (32 / baseMarker.scale / scaleFactor).toFixed(1);
```

Comment:
I don't really have a problem clicking them, but I guess that varies to the user. It makes it equal to how clickable icons are on FA so making that consistent. I did find it annoying trying to click planes that are nearly on top of each with the transparent border. But it can be added back in if it seems like it should be